db-migrations/25_message_varchar_length.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE version_endpoints DROP COLUMN message;
```

Comment:
This might cause data loss, preferably we could just trim the content, e.g:

```
ALTER TABLE version_endpoints ALTER COLUMN message varchar(64)
USING SUBSTR(message, 1, 64)
```